app/globals.css

```
@@ -375,3 +375,558 @@ h6 {
+    margin: 0;
+    padding: 0;
+    overflow-x: hidden;
+    /* Remove position fixed to allow scrolling */
+  }
+}
+
+/* PWA Mode - Complete URL Bar Removal WITH SCROLLING */
+@media (display-mode: standalone), (display-mode: fullscreen) {
+  html, body {
+    height: 100vh !important;
+    height: -webkit-fill-available !important;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)